### PR TITLE
Updated to infinispan-operator 1.0.0

### PR DESCRIPTION
**Pull request by rigazilla**
Updated infinispan-operator tutorial to operator 1.0.0 Infinispan 10.x

### CODE REVIEW

File: operator/java-client/src/main/java/org/infinispan/App.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        cache.put("key", "value");
+        // Retrieve the value and print it out
+        System.out.printf("key = %s\n", cache.get("key"));
+        // Stop the cache manager and release all resources
+        cacheManager.administration().removeCache("quickstart-cache");
+        cacheManager.stop();
+    }
+
+    static void configure(String args[]) {
+        Getopt g = new Getopt("testprog", args, "h:p:U:P:");
```
Comment (galderz) [inline operator/java-client/src/main/java/org/infinispan/App.java]: The code would be simpler if you just used system properties for the different parameters you want to pass in. It'd also remove a dependency. The focus here is not so much the validation part, but showing that you can just use plain Java client code to talk to Infinispan started by operator.

Also, given that authentication is enabled by default, I'd check that user and password system properties are given by the user.

Eventually, the development profile would avoid the need for username and password for the simplest of example, but you can keep a very simple separate example just for that.

File: operator/README.md [hunk 2]
```diff
@@ -77,53 +76,93 @@ example-infinispan-<id_3>]
...
+Hot Rod is a custom binary TCP protocol designed for Infinispan client/server interaction. Hot Rod clients are available in several programming languages but this tutorial provides a Java example for demonstration purposes.
+
+1. Change to the `java-client` directory.
+```bash
+cd java-client
+```
+
+2. Run the Hot Rod Java client as follows:
+```bash
+mvn exec:java -Dexec.mainClass="org.infinispan.App" -Dexec.args="-h 172.30.125.152 -U developer -P ${PASS}"
```
[the remark this answers is not in the document]
Comment (rigazilla) [reply operator/README.md]: Yeah oversight. The way `INFINISPAN_HOST` is populated needs a reviews, now it works only if LoadBalancer can expose external IP.

File: operator/README.md [hunk 3]
```diff
@@ -77,53 +76,93 @@ example-infinispan-<id_3>]
...
+credentials:
+  - username: developer
+     password: n4Jx@Inm9IaNpQ8a
+  - username: operator
+     password: vsujrixeZ6kXsOj5
+```
+
+3. Export the password for the `developer` user to a local variable.
+```bash
+export PASS=n4Jx@Inm9IaNpQ8a
```
[the remark this answers is not in the document]
Comment (rigazilla) [reply operator/README.md]: I can add the yq filter to the command, but I would also leave the manual procedure for who want to understand what's happening without decoding the bash command.

File: operator/java-client/src/main/java/org/infinispan/App.java (hunk 1 above)
Comment (rigazilla) [reply operator/java-client/src/main/java/org/infinispan/App.java]: System vars sound like a good idea, will do.
I need this branch stable atm, will do change later.

File: operator/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (galderz) [reply operator/README.md]: Btw, the `yq` command line tool I use is [this](https://github.com/kislyuk/yq).

**Comment by karesti** [comment]
closed for #87

**Pull request closed without merging**